SPECS/xo-lite.spec

```diff
@@ -0,0 +1,31 @@
+Summary: Xen Orchestra Lite
+Name:    xo-lite
+Version: 0.1.3
+Release: 1%{?dist}
+License: AGPL3-only
+URL:     https://github.com/vatesfr/xen-orchestra
+
+BuildArch: noarch
+
+Source0: https://github.com/vatesfr/xen-orchestra/releases/download/%{name}-v%{version}/%{name}-%{version}.tar.gz
+
+%description
+This package contains Xen Orchestra Lite, a lightweight version of the Xen
+Orchestra Appliance for single-host administration, running directly from your
+browser without having to deploy anything.
+
+%prep
+%autosetup -p1
+
+%install
+%{__install} -d -m 755 %{buildroot}/opt/xensource/www
+%{__cp} -a * %{buildroot}/opt/xensource/www
+
+%files
```

Comment:
It would be good to have a LICENSE file in the tarball, that we could embed in the RPM with `%license`.

(note that if this happens your `cp -a *` above will be too broad)

Reply:
Fixed